### PR TITLE
Convert ostree_boot_image task to a container

### PR DESCRIPTION
**Pull request by p3ck**
The artifact.ci.centos.org forwarding should be done with a
proper http proxy instead.  But this works now.

### CODE REVIEW

File: tasks/ostree-boot-image [hunk 1]
```diff
@@ -1,10 +1,23 @@
...
 env
 
 # Find the base dir
 base_dir="$(dirname $0)/.."
+HOMEDIR=$(pwd)
+
+function clean_up {
+    set +e
+    # Tear down the atomic host and image
+    sudo docker run --privileged -it --link libvirtd:libvirtd -e commit=${commit} -e image2boot=${image2boot} -e state=absent -v /var/lib/libvirt/:/var/lib/libvirt/ -v ${HOMEDIR}/output/:/root/output/ -v ${basedir}/:/root/ci-pipeline ostree_boot_image
```
Comment (johnbieren) [inline tasks/ostree-boot-image]: So this is the part that I think will not be part of this stage and will instead come after the integration and package level tests once this is put into container() calls in the Jenkinsfile

File: config/Dockerfiles/ostree_boot_image/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+COPY atomic7-testing.repo /etc/yum.repos.d
+COPY walters-buildtools.repo /etc/yum.repos.d
+
+RUN yum -y install libguestfs libguestfs-tools-c \
+                   git ostree rpm-ostree libvirt-client \
+                   PyYAML python2-setuptools virt-install \
+                   python-pip python-devel gcc net-tools \
+                   openssh-clients sshpass && yum clean all
+
+RUN pip install ara
```
Comment (johnbieren) [inline config/Dockerfiles/ostree_boot_image/Dockerfile]: Do you want to version lock this like it is here? https://github.com/CentOS-PaaS-SIG/ci-pipeline/blob/master/playbooks/roles/ara/tasks/main.yml

File: tasks/ostree-boot-image (hunk 1 above)
Comment (p3ck) [reply tasks/ostree-boot-image]: agreed.  But for now we can do it here.

File: config/Dockerfiles/ostree_boot_image/Dockerfile (hunk 2 above)
Comment (p3ck) [reply config/Dockerfiles/ostree_boot_image/Dockerfile]: Running on the latest centos7.4 it won't work with the version locked at 13.

Comment (arilivigni) [reply config/Dockerfiles/ostree_boot_image/Dockerfile]: We discussed this and in the future the staging pipeline would catch this so it shouldn't be a concern not to pin it

**Pull request merged**